### ISSUE TITLE
As fallback create title from filename if no other metadata is available.

### ISSUE THREAD

**Issue by titusz** [iscc/iscc-specs#81]
We should consider to supply the filename (maybe stripped of file extension) as seed metadata for Meta-ID creation if no other metadata can be acquired. In any case the order of operations to acquire seed metadata during automated ISCC creation should be part of the specification.

**Comment by lrosenthol** [comment]
You need to be *very careful* about this from a PII perspective, as filenames are frequently loaded with information that may be personalized and could expose unwanted information.

**Comment by titusz** [comment]
Yes that is an issue. But it is an issue for any kind of user generated metadata. We should add a warning in the specification. Something along the lines of:

"Implementations that support fully automated ISCC generation and expose both the ISCC and auto-generated metadata publicly must warn its users about potential privacy risks and require manual confirmation."
